lib/Test2/Event/Encoding.pm

```diff
@@ -0,0 +1,17 @@
+package Test2::Event::Encoding;
```

Comment:
I will make a PR for the test-more repo to add this event and the `TAP::Version` event.